### DISCUSSION TITLE
How to add a email list

### DISCUSSION THREAD

**Discussion by TanYuanXiangElroy** [alshedivat/al-folio#2097] [Ideas]
propose the inclusion of an email list feature in the website template

an email list can serve as a powerful tool to enhance communication and engagement

for stuff like Timely Updates, Community Building, Event Announcements, Exclusive Content, Research Collaboration

**Comment by george-gca** [comment]
What do you mean exactly? Something like asking for people's emails to create a newsletter?

**Reply by TanYuanXiangElroy** [reply]
Something like that

**Reply by george-gca** [reply]
I have no idea how to do that, or even if it would be useful for more people. But feel free to send a PR if you manage to implement this, maybe it could be something that more users would benefit from.

**Comment by george-gca** [comment]
@TanYuanXiangElroy take a look at [this template](https://horsfall.dev/jekyll-startbootstrap-agency/). It might be useful to check [how they done it](https://github.com/davehorsfall/jekyll-startbootstrap-agency).

**Comment by iamindra7** [comment]
### if you want to add an email list of you / your team's  you can do that using html list tag ..
### and if you want email input field of the user you can do that by creating input fields....

**Comment by george-gca** [comment]
Implemented in #2517.
